backend/routes/userRoutes.js

```diff
@@ -11,5 +11,7 @@ const router = express.Router();
 router.route("/").get(protect, allUsers);
 router.route("/").post(registerUser);
 router.post("/login", authUser);
+router.post("/verify-email", verifyEmail);
+router.post("/forgot-password", forgotPassword);
```

Comment:
Hey I think for forget password the issue is there #2 put for mail confirmation hmm.. wait let me see

Reply:
@praveenarjun Please add a short video of the changes u made and along with it's woking,  seeing fine with the code but with forget password having second thoughts